### DISCUSSION TITLE
自定义组件在form中如何触发校验

### DISCUSSION THREAD

**Discussion by chen-tq** [element-plus/element-plus#9332] [Q&A]
我写了一个自定义组件<my-input>，我希望在onblur或onchange时，能触发el-form-item的rule校验，应该怎么实现？

如下例：
`
<el-form ref="ruleFormRef" :model="form" :rules="rules">
    <el-form-item label="自定义输入框" prop="myinput" :rules="[{required: true, message: '不能为空', trigger:'blur'}]">
        <my-input v-modle="myinput" />
    </el-form-item>
</el-form>
`

**Comment by TLovers** [comment]
validateField. 这个方法不行吗？ 失去焦点的时候调用

**Comment by abigmiu** [comment]
```javascript
const elFormItem = inject(formItemContextKey);
if (elFormItem) {
     elFormItem.clearValidate();
 }
```
